src/Controller/CaseController.php

```diff
@@ -633,4 +646,33 @@ public function move(BoardRepository $boardRepository, CaseEntity $case, Request
             'case' => $case,
         ]);
     }
+
+    /**
+     * @Route("/{id}/delete", name="case_delete", methods={"POST"})
+     */
+    public function delete(CaseEntity $case, Request $request): Response
+    {
+        $this->denyAccessUnlessGranted('edit', $case);
+
+        $deleteForm = $this->createForm(CaseDeleteType::class, $case);
+
+        $deleteForm->handleRequest($request);
+
+        if ($deleteForm->isSubmitted() && $deleteForm->isValid()) {
+            $case->setSoftDeleted(true);
+            $now = new \DateTime('now');
+            $case->setSoftDeletedAt($now);
```

Comment:
It's weird that `SoftDeletableEntity` has both a `softDeleted` and a `softDeletedAt` property. `softDeletedAt` should enough, but we should fix that in another branch.

Reply:
Yeah let's fix this on another branch as it will require some code changes in various other sections.